database/feegrant.go

```diff
@@ -0,0 +1,46 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant"
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// SaveFeeGrantAllowance allows to store the fee grant allowances for the given block height
+func (db *Db) SaveFeeGrantAllowance(allowance *feegranttypes.MsgGrantAllowance, height int64) error {
+	stmt := `
+INSERT INTO fee_grant_allowance(grantee, granter, allowance, height) 
+VALUES ($1, $2, $3, $4) 
+ON CONFLICT DO NOTHING`
+
+	allowanceJSON, err := codec.ProtoMarshalJSON(allowance.Allowance, nil)
+	if err != nil {
+		return fmt.Errorf("error while marshaling grant allowance: %s", err)
+	}
+
+	// Store the accounts
+	var accounts []types.Account
+	accounts = append(accounts, types.NewAccount(allowance.Grantee), types.NewAccount(allowance.Granter))
+	err = db.SaveAccounts(accounts)
+	if err != nil {
+		return fmt.Errorf("error while storing fee grant allowance accounts: %s", err)
+	}
+
+	_, err = db.Sql.Exec(stmt, allowance.Grantee, allowance.Granter, allowanceJSON, height)
+	if err != nil {
+		return fmt.Errorf("error while storing fee grant allowance: %s", err)
+	}
+
+	return nil
+}
+
+// RevokeFeeGrantAllowance removes the fee grant allowances data from the database
+func (db *Db) RevokeFeeGrantAllowance(grantee string, granter string) error {
```

Comment:
Since BDJuno can be handling multiple blocks at the same time, we also need to make sure we pass the `height` to the delete methods in order to make sure we delete the rows only for older heights, and not for newer ones that might have been parsed at the same time. For this reason, I think we can define a new type that is 

```go
type GrantRemoval struct {
  Grantee string
  Granter string
  Height int64 
}
```

and then we can pass a single `grantDeletion` to the `RevokeFeeGrantAllowance` method.  

Also, this method should be named `DeleteFeeGrant` instead cause it might be used elsewhere (eg. when a feegrant expires or when the user uses all the allowance he had available).